### ISSUE TITLE
Receiving unexpected ErrInvalidWebhookURLResponseText

### ISSUE THREAD

**Issue by MichaelUrman** [atc0005/go-teams-notify#197]
I'm seeing logs with an error of the following format:
> ...: got "{\"version\":\"1.1\",\"content\":{\"headers\":[{\"key\":\"Content-Type\",\"value\":[\"text/plain; charset=utf-8\"]}]},\"statusCode\":200,\"reasonPhrase\":\"OK\",\"headers\":[],\"trailingHeaders\":[],\"requestMessage\":null,\"isSuccessStatusCode\":true}", expected "1": invalid webhook URL response text

This appears to derive from the following check, which references #59 and by relation the fix #81. This seems to suggest that `1` is no longer the only acceptable response body, as these logs correspond with messages that successfully made it to Teams.

https://github.com/atc0005/go-teams-notify/blob/c205f3b9c9e4b371106e80d00e48083bbe7eee45/send.go#L404-L410

Following the hints in https://github.com/MicrosoftDocs/msteams-docs/issues/1546#issuecomment-634273664 I am able to reproduce this with cURL. It similarly claims that as of 30 months ago `curl -H 'Content-Type: application/json' -d '{"text": "Hello World"}' <YOUR WEBHOOK URL>` would display 1, but I get the following json instead:

```json
{"version":"1.1","content":{"headers":[{"key":"Content-Type","value":["text/plain; charset=utf-8"]}]},"statusCode":200,"reasonPhrase":"OK","headers":[],"trailingHeaders":[],"requestMessage":null,"isSuccessStatusCode":true}
```

(FWIW, in case version matters, I'm actually encountering this while using go-teams-notify v2.6.1. I'm not blocked on this, but I'd like to clean up my logs.)

**Comment by MichaelUrman** [comment]
Closing as Microsoft considered this a bug and have restored the `1` response body. Per the comments on MicrosoftDocs/msteams-docs \#7830 this has occurred before, so I'll leave it up to you whether you'd want to attempt to handle the undocumented response. (It appears to contain enough information to properly diagnose success vs error, but as it's not documented it's a risk.)

**Comment by atc0005** [comment]
@MichaelUrman Thanks for filing this GH issue.

Apologies for the late response. For whatever reason I either missed the notification from GitHub that an issue was opened in this repo or looked right past it.

Thank you for the detailed report and the various references for further information.

I'm now following both of these GH issues for further activity:

- https://github.com/MicrosoftDocs/msteams-docs/issues/7830
- https://github.com/MicrosoftDocs/msteams-docs/issues/7852

Please reopen or file another issue here as appropriate if you spot further issues.
